src/NuGet/BuildNuGets.csx

```diff
@@ -77,6 +77,12 @@ var IsCoreBuild = File.Exists(Path.Combine(ToolsetPath, "corerun"));
 var NuGetAdditionalFilesPath = Path.Combine(SolutionRoot, "build/NuGetAdditionalFiles");
 var ThirdPartyNoticesPath = Path.Combine(NuGetAdditionalFilesPath, "ThirdPartyNotices.rtf");
 var NetCompilersPropsPath = Path.Combine(NuGetAdditionalFilesPath, "Microsoft.Net.Compilers.props");
+var NetcoreCscPropsPath = Path.Combine(NuGetAdditionalFilesPath, "Microsoft.NETCore.Csc.props");
```

Comment:
Should we just pass a single variable "NuGetAdditionalFiles" to the nuget pack invocation at this point?